airbyte-integrations/connectors/source-relational-db/src/main/java/io/airbyte/integrations/source/relationaldb/state/GlobalStateManager.java

```diff
@@ -99,6 +102,26 @@ private CdcState extractCdcState(final AirbyteStateMessage airbyteStateMessage)
     }
   }
 
+  private Set<AirbyteStreamNameNamespacePair> extractStreams(final AirbyteStateMessage airbyteStateMessage) {
```

Comment:
@subodh1810 It would be nice to add some tests to `GlobalStateManagerTests` that verifies the behavior when global or legacy for this method and `convertDbStreamSTateToAirbyteStreamState`